src/Rector/FuncCall/DispatchNonShouldQueueToDispatchSyncRector.php

```diff
@@ -92,20 +100,17 @@ public function refactor(Node $node): FuncCall|MethodCall|StaticCall|null
 
     private function processCall(FuncCall|MethodCall|StaticCall $call): FuncCall|MethodCall|StaticCall|null
     {
-        static $shouldQueueType = new ObjectType('Illuminate\Contracts\Queue\ShouldQueue');
-
         if (! $call->args[0] instanceof Arg) {
             return null;
         }
 
+        $objectType = new ObjectType(self::SHOULD_QUEUE_INTERFACE);
```

Comment:
could probably be a static var

```suggestion
        static $objectType = new ObjectType(self::SHOULD_QUEUE_INTERFACE);
```